<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>顶点操作</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>待增加或删除的顶点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="vtNumber"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="vtAddBtn">
            <property name="text">
             <string>添加顶点</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="vtRemoveBtn">
            <property name="text">
             <string>删除顶点</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>边操作</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>顶点A</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="vertexUInput">
               <property name="font">
                <font>
                 <family>Cascadia Code</family>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>顶点B</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="vertexVInput">
               <property name="font">
                <font>
                 <family>Cascadia Code</family>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>权重</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="weightInput">
               <property name="font">
                <font>
                 <family>Cascadia Code</family>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addEdgeButton">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>添加新边</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeEdgeButton">
            <property name="text">
             <string>删除边</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="adjListDisplay">
      <property name="font">
       <font>
        <family>Cascadia Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>423</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>请创建图，邻接链表将在上方框中显示，点击“确定”进行后续操作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="judge">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
